docs/onboarding_toolkit.md

```diff
@@ -1 +1,172 @@
-#  Onboarding toolkit
+# Ansible Project Onboarding
+
+Welcome to the Ansible ecosystem of projects! This document guides you 
+through the steps to take to become part of the Ansible ecosystem.
+
+# Why be part of the Ansible ecosystem
+Being a part of the Ansible ecosystem means you are tapping into a large and
+thriving community of passionate people working and volunteering in Ansible.
+
+The major benefits of becoming part of the Ansible ecosystem include:
+
+* A presence on docs.ansible.com/ecosystem for your project documentation,
+ as well as being part of the official Ansible ReadTheDocs space.
+* A presence on the (future) Ansible.com to introduce your project.
+* Dedicated communications channels (chat and forum) to communicate with your
+community and attract other community members to your project.
+
+
+
+# Setting up your GitHub repository
+
+The Ansible ecosystem projects exist in the following two GitHub repositories:
```

Comment:
Should we consider ansible-collections here as well? Or at least mention that collections have their own space and are not covered here?